### DISCUSSION TITLE
How do you do drag & drop?

### DISCUSSION THREAD

**Discussion by iamursky** [ueberdosis/tiptap#1519] [General Questions]
Hey there,

Ho do you add drag & drop support in your projects?
What possible solutions are even exist to add it?

I found this [gist](https://gist.github.com/slava-vishnyakov/16076dff1a77ddaca93c4bccd4ec4521), but the problem I see here is that we have to write own extension from ground up and not extending an existing one loosing community updates (bugfixes/features).

**Comment by DioNNiS** [comment]
I need a drop file into the editor as well
